packages/ui/src/tmp/tests/selectors/home.selectors.ts

```diff
@@ -0,0 +1,15 @@
+export enum HomePageSelectors {
+  AssetList = 'asset-list',
+  HomePageContainer = 'home-page-container',
```

Comment:
Why do we need to move the test selectors to the `ui` package? Can't the actual selectors just be relocated off the ui component directly? Will those be consistent b/w web and mobile? I don't think all of these selectors are used on ui components?

Reply:
I just dumped the list here as some of them are used in UI components. I can curate it to only have what we need specifically in this package but I didn't want to spend too long on it if I'm not sure what to do. 

We need them both in the extension in the test code and here in the UI components. I added it to the [discussion](https://github.com/orgs/leather-wallet/discussions/66#discussioncomment-9595101) but I didn't want it to block this work.

We could:
- put them in a shared place like `utils` / a new test package
- pass them into the UI components 
- refactor our other code to use selectors on code that is in the extension and not these components